lib/matplotlib/projections/geo.py

```diff
@@ -190,7 +190,7 @@ def set_longitude_grid(self, degrees):
         """
         Set the number of degrees between each longitude grid.
         """
-        number = (360.0 / degrees) + 1
+        number = int(360.0 / degrees) + 1
```

Comment:
`360 / degrees`, no need for `.0` anymore (... especially given that you're casting to int immediately after).  Same below.

Also I have no knowledge about this part of the code but are you sure it's shouldn't be `math.ceil(360/degrees)`?  (Just guessing.)